mlir/include/mlir/Dialect/MemRef/IR/MemRefOps.td

```diff
@@ -1351,7 +1351,11 @@ def MemRef_PrefetchOp : MemRef_Op<"prefetch"> {
     instruction cache.
   }];
 
-  let arguments = (ins AnyMemRef:$memref, Variadic<Index>:$indices,
+  // The memref argument is labeled with a MemWrite side effect to enforce a
+  // relative ordering of the prefetch and other memory operations targeting
+  // that memory stream.
```

Comment:
```suggestion
  // The memref argument is labeled with a side effect to enforce a
  // relative ordering of the prefetch and other memory operations targeting
  // that memory stream.
  // We need it to be a write otherwise the operation would be trivially removed
  // since it does not produce a value.
```

Reply:
Thanks for the wording, implemented the suggested wording change.